Python/Product/Profiling/Profiling/StandaloneTargetView.cs

```diff
@@ -166,6 +166,7 @@ public StandaloneTargetView(IServiceProvider serviceProvider, StandaloneTarget t
                     _interpreterPath = value;
                     if (Interpreter.Name == Strings.LaunchProfiling_OtherInterpreter && Interpreter.Path == null) {
                         Interpreter = new PythonInterpreterView(Strings.LaunchProfiling_OtherInterpreter, Strings.LaunchProfiling_OtherInterpreter, _interpreterPath);
+                        CanSpecifyInterpreterPath = (_interpreter.Name == _customInterpreter.Name);
```

Comment:
In the setter for `Interpreter`, a similar check is written as `(_interpreter == _customInterpreter)`. I think the intent was to reuse `_customInterpreter` object in this scenario, rather than creating a new view every time.